### PR TITLE
This make our resource-example.xml validate.

### PR DESCRIPTION
**Pull request by msdemlei**
Specifically, I'm removing a few suprefluous xmlns declarations,
fix schemaLocation to point to the vs namespace actually used,
and make the "more columns" thing a comment.

This now passes validation with a plain xerces (and of course with
dachs adm xsd.

### CODE REVIEW

**Review by mbtaylor — DISMISSED** [review]
I don't see the xmlns:oai namespace doing work - can that be removed too?

**Comment by jesusjuansalgado** [comment]
> I don't see the xmlns:oai namespace doing work - can that be removed too?

Waiting for an answer from Markus before the merging

**Review by mbtaylor — APPROVED** [review]

**Comment by jesusjuansalgado** [comment]
Thanks both!

**Comment by msdemlei** [comment]
On Tue, Apr 13, 2021 at 09:01:07AM -0700, Mark Taylor wrote:
> @mbtaylor approved this pull request.
> 
> I don't see the xmlns:oai namespace doing work - can that be removed too?

Of course, and that's actually a good idea because in publishing
registries the oai namespace will in general already be declared in
the container, and while I don't expect it to be bound to anything
else ever, not repeating it takes away another source of confusion.

I've updated the PR.

While I'm talking: I still believe the whole thing should be taken
out of the main document using ivoatex's auxiliaryurl feature.

However, if you want to keep it in-line, save a tree and change

  \lstinputlisting[language=XML]{resource-example.xml}

to

  \lstinputlisting[language=XML,basicstyle=\footnotesize]{resource-example.xml}

(i.e., make the font a bit smaller).

**Pull request merged**
